test/get_authors_test.rb

```diff
@@ -0,0 +1,36 @@
+require "minitest/autorun"
+require "fileutils"
+
+class TestGetAuthors < Minitest::Test
+  def setup
+    @original_dir = Dir.pwd
+    @dir = Dir.mktmpdir 'developer-portal'
+
+    @subdir = File.join(@dir, 'subdir')
+
+    Dir.mkdir @subdir
+
+    FileUtils.cp './test/fixtures/README.md', @subdir
+
+    Dir.chdir @dir
+
+    `git config --global user.email "you@example.com"`
```

Comment:
This configuration of git could be done as function for use in tests.
Also if a person runs tests in work environment the personal configurations get overwritten.

Reply:
Right, I run this on my working machine a lose my auth. Maybe it can be set for this repo only by just omitting `--global`?

Thanks for spotting this.